core/mgmt/v1beta/metric.proto

```diff
@@ -158,3 +158,36 @@ message ListPipelineTriggerChartRecordsResponse {
   // A list of pipeline trigger records.
   repeated PipelineTriggerChartRecord pipeline_trigger_chart_records = 1;
 }
+
+// CreditConsumptionChartRecord contains credit consumption metrics, aggregated
+// by owner and time frame.
+message CreditConsumptionChartRecord {
+  // Credit owner UUID.
+  string credit_owner_uid = 1;
+  // Time buckets.
+  repeated google.protobuf.Timestamp time_buckets = 2 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Total credit consumed in each time bucket.
+  repeated float amount = 3 [(google.api.field_behavior) = OUTPUT_ONLY];
+}
+
+// ListCreditConsumptionChartRecordsRequest represents a request to list pipeline
+// trigger metrics, aggregated by pipeline ID and time frame.
+message ListCreditConsumptionChartRecordsRequest {
+  // Aggregation window. The value is a positive duration string, i.e. a
+  // sequence of decimal numbers, each with optional fraction and a unit
+  // suffix, such as "300ms", "-1.5h" or "2h45m".
+  string aggregation_window = 1;
```

Comment:
Watch out @EiffelFly 👀 This is different than in pipelines (where duration is provided in nanoseconds) but I think it makes for a clearer API.

Reply:
This is indeed clearer 🥳 But I am wondering do we plan to unify this kind of difference between different metric endpoint soon? cc @donch1989

Reply:
I agree we need to make the style of each metric endpoint consistent. We can note this down and plan it for later.